### DISCUSSION TITLE
Why is @nx/angular used as a dependency and not as a devDependency?

### DISCUSSION THREAD

**Discussion by Kaemmelot** [nrwl/nx#22381] [Feature Requests]
When creating a new Nx Angular monorepo, `@nx/angular` is added as production dependency in the `package.json` even though it seems that it does not provide any production code. Why is that?
Shouldn't it be in the devDependencies section instead?

**Comment by bulldog98** [comment]
If I remember correctly there is a nx ngrx feature generation for angular availabe, and that generates code, that imports functions from @nx/angular. Not sure if that is still the case.
